lang/user/id.xml

```diff
@@ -369,21 +362,15 @@ Jangan matikan sistem atau tutup aplikasi ini.</warning_saving>
 	</game_data>
 
 	<home_screen>
-		<filter>Filter</filter>
+		<filter>Penyesuaian</filter>
 		<sort_app>Urutan Aplikasi dari</sort_app>
 		<all>Semua</all>
 		<by_region>Dari Region</by_region>
-		<usa>USA</usa>
 		<europe>Europa</europe>
 		<japan>Jepang</japan>
-		<asia>Asia</asia>
 		<by_type>Dari Type</by_type>
 		<commercial>Komersial</commercial>
-		<homebrew>Homebrew</homebrew>
```

Comment:
<homebrew>Paket Game</homebrew>

Reply:
No need change, homebrew include homebrew apps/games

Reply:
no need both here, also using the default string

Reply:
In Malaysia, it's just the default, but in Indonesia, if you can just use the words game package, it means the same as homebrew

Reply:
but i think is no good, homebrew not just games also include third-party apps

Reply:
Yes, I know, but Paket game means the same in Indonesian

Reply:
But I think it should refer to applications, not just games

Reply:
Oh, if that's the case, what if that means shop in Indonesian, toko paket so what does it mean for games or applications?

Reply:
Oh, if that's the case, what if that  toko paket means shop in Indonesian, so what does it mean for games or applications?

Reply:
I think you can just leave "homebrew" as is, paket game is "game package/bundle" which is different thing.